lighthouse-core/report/html/report-styles.css

```diff
@@ -1105,6 +1123,16 @@
   --gauge-circle-size: var(--gauge-circle-size-big);
 }
 
+.lh-sticky-header--visible > .lh-gauge__wrapper--fail > .lh-gauge__svg-wrapper {
+  background-image: url("data:image/svg+xml,%3Csvg width='36' height='36' viewBox='0 0 36 36' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M17 26H20V23H17V26Z' fill='%23FF4E42'/%3E%3Cpath d='M17 20H20V11H17V20Z' fill='%23FF4E42'/%3E%3C/svg%3E");
+  height: 36px;
+  width:36px;
+}
+
+.lh-category-header .lh-gauge__wrapper--fail  .lh-gauge__svg-wrapper{
```

Comment:
we don't want this to appear with all failing scores, just errored ones

![image](https://user-images.githubusercontent.com/2301202/99421983-98f7a680-28c4-11eb-92cc-cfa9385ea054.png)
 
(what it looks like right now)

Reply:
For guage, I was only able to find these classes for guage `.lh-gauge__wrapper--pass` `.lh-gauge__wrapper--average` `.lh-gauge__wrapper--fail` or `.lh-gauge__wrapper--not-applicable`
There doesn't seem to any class for error, Do I need to add it for `not-applicable` one?
Thank you so much for answering my noob questions :)

Reply:
Looks like you'll need to add one in the same place where you updated `percentageEl.textContent` :)